compiler/test/grainfmt/application.expected.gr

```diff
@@ -85,3 +85,5 @@ let preExistingObjectsWithRefCountMismatch2 = Map.make():
   >
 
 Int32.(-)(1l, 2l)
+Int32.(+)(x=1l, y=2l)
```

Comment:
In this example wouldn't the code path you edited not get applied? For example the line directly above this is formatted as `Int32.(-)(1l, 2l)`. Would you want to `use Int32.{ (+) }` first?

Reply:
100%, I added the correct tests, I kept these ones around for completeness sake.